howtos/documentation-guidelines.md

```diff
@@ -162,6 +162,43 @@ If you wish to test `.htaccess` rules, you have a couple options:
 
    4. Clean up the server with `docker compose down`.
 
+## Screenshot automation
+
+In an effort to automate screenshots across Camunda Platform 8 documentation, the following teams execute uniform steps when incorporating images and diagrams:
+
+:::note
```

Comment:
@akeller Would you word this differently?

Reply:
I'm good with this! We can always iterate 😄